services/sidecar/src/simcore_service_sidecar/core.py

```diff
@@ -174,12 +174,15 @@ async def inspect(
     run_result = SUCCESS
     next_task_nodes = []
     try:
-        sidecar = Executor(
+        executor = Executor(
             db_engine=db_engine, rabbit_mq=rabbit_mq, task=task, user_id=user_id,
         )
-        await sidecar.run()
+        await executor.run()
         next_task_nodes = list(graph.successors(node_id))
-    except (aiodocker.exceptions.DockerError, exceptions.SidecarException):
+    except asyncio.CancelledError:
+        run_result = FAILED
+        log.warning("Task has been cancelled")
```

Comment:
for all this type of warning logs ... we might want to see more info if log level is set to debug. Perhaps we can
```python
   log.warning("task has been cancelled", stack_info =  SIDECAR_LOGLEVEL == logging.DEBUG)
```
This way, in production we can notice the exception and by turning to debug we could get more info about it, i.e. why it happened

We could generally apply this

Reply:
cool, did not know about that stack_info thing

Reply:
> cool, did not know about that stack_info thing
yes :-)

Reply:
added